### ISSUE TITLE
Keepass 2.6.6 doesn't open 

### ISSUE THREAD

**Issue by davidrojo** [keepassxreboot/keepassxc#6772]
## Overview
I cannot open keepass. It is installed via snap, and has been working fine.

I think this is happening since the last snap update.

I've tried to uninstall and reinstall, but it still does not work.

## Steps to Reproduce
1. run keepass

## Expected Behavior
The application opens

## Actual Behavior
The application does not open, doesn't show any error, but the process is running.

## Context
```
$ keepassxc
Gtk-Message: 16:56:06.552: Failed to load module "xapp-gtk3-module"
(keepassxc:9250): dbind-WARNING **: 16:56:06.555: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-eeQf572JNN: No such file or directory
(keepassxc:9250): Gtk-WARNING **: 16:56:06.558: Theme parsing error: gtk.css:1566:23: 'font-feature-settings' is not a valid property name
(keepassxc:9250): Gtk-WARNING **: 16:56:06.561: Theme parsing error: gtk.css:3618:25: 'font-feature-settings' is not a valid property name
(keepassxc:9250): Gtk-WARNING **: 16:56:06.561: Theme parsing error: gtk.css:4080:23: 'font-feature-settings' is not a valid property name
Gtk-Message: 16:56:06.598: Failed to load module "canberra-gtk-module"
Gtk-Message: 16:56:06.599: Failed to load module "canberra-gtk-module"
Qt: Session management error: Could not open network socket
QPainter::begin: Paint device returned engine == 0, type: 3
QPainter::save: Painter not active
QPainter::setPen: Painter not active
QPainter::setBrush: Painter not active
QPainter::setRenderHint: Painter must be active to set rendering hints
QPainter::setRenderHint: Painter must be active to set rendering hints
QPainter::opacity: Painter not active
QPainter::brush: Painter not active
QPainter::pen: Painter not active
QPainter::setPen: Painter not active
QPainter::setOpacity: Painter not active
QPainter::drawPath: Painter not active
QPainter::setPen: Painter not active
QPainter::setOpacity: Painter not active
QPainter::setBrush: Painter not active
QPainter::drawPath: Painter not active
QPainter::setBrush: Painter not active
QPainter::setOpacity: Painter not active
QPainter::restore: Unbalanced save/restore
QPainter::end: Painter not active, aborted
```

# Keepass version
KeePassXC - 2.6.6

# System
Operating System: Linux (Ubuntu 21.04)
Desktop Env: Gnome
Windowing System: ?

**Comment by droidmonkey** [comment]
Please clarify: 
Did KeePassXC 2.6.6 work for you previously and the only thing that changed was an update to snap itself?

**Comment by davidrojo** [comment]
I really don't know if the version 2.6.6 has worked at any time as snap is automatically updated.

I've checked another computer with ubuntu also, that I had version 2.6.4, it has been updated to 2.6.6 and works, but in my primary computer it doesn't.

This is the log I receive in the other computer where it works.

```
$ keepassxc
(keepassxc:27335): Gtk-WARNING **: 08:48:37.938: Theme parsing error: gtk.css:1566:23: 'font-feature-settings' is not a valid property name
(keepassxc:27335): Gtk-WARNING **: 08:48:37.942: Theme parsing error: gtk.css:3618:25: 'font-feature-settings' is not a valid property name
(keepassxc:27335): Gtk-WARNING **: 08:48:37.943: Theme parsing error: gtk.css:4080:23: 'font-feature-settings' is not a valid property name
Gtk-Message: 08:48:38.010: Failed to load module "canberra-gtk-module"
Gtk-Message: 08:48:38.011: Failed to load module "canberra-gtk-module"
Qt: Session management error: Could not open network socket
```

I shows also the error loading module canberra-gtk-module but not the other errors:

```
Gtk-Message: 16:56:06.552: Failed to load module "xapp-gtk3-module"
(keepassxc:9250): dbind-WARNING **: 16:56:06.555: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-eeQf572JNN: No such file or directory
```
I gues the problem shoud be one of these, but I don't know how to fix it

**Comment by phoerious** [comment]
Please try a clean installation of the snap. This looks like some external error.

**Comment by davidrojo** [comment]
I've removed the installed package from snap, and reinstalled. Same issue, Keepass doesnt open although the process is running in background.

I've installed keepass through the distribution package (sudo apt install keepassxc) and it works fine, although the version installed is the 2.6.2 not the 2.6.6

I've downloaded the .AppImage file from https://keepassxc.org/download/#linux and it works  (version 2.6.6).

So, the problem is the version downloaded via snap.
